OTRAssets/Strings/OTRStrings.h

```diff
@@ -10,6 +10,8 @@ FOUNDATION_EXPORT NSString* ABOUT_VERSION_STRING();
 FOUNDATION_EXPORT NSString* ACCOUNTS_STRING();
 /** "Account", The string describing a buddy's account */
 FOUNDATION_EXPORT NSString* ACCOUNT_STRING();
+/** "Active", segmented control value for active friends */
+FOUNDATION_EXPORT NSString* ACTIVE_BUDDIES_STRING();
```

Comment:
@chrisballinger Yes, I wondered about that, but decided against reuse because the comment says something about "Will be shown in the UI as Active x ago".